app/views/admin/users/index.html.haml

```diff
@@ -1,10 +1,14 @@
 .row
   .col-md-12
     .page-header
-      %h2
+      %h1
         Users
         - if @users
           = "(#{@users.length})"
+
+        - if can? :create, User && ENV['OSEM_ICHAIN_ENABLED'] != 'true'
```

Comment:
Please make the addition in the models/ability.rb then you don't need `&& ENV['OSEM_ICHAIN_ENABLED'] != 'true'` here or the if statement in the controller admin/users#create.